global.json

```diff
@@ -1,5 +1,5 @@
 {
-  "msbuild-sdks": {
-    "MSBuild.Sdk.Extras": "1.6.61"
-  }
+    "msbuild-sdks": {
+        "MSBuild.Sdk.Extras": "2.0.41"
```

Comment:
Why not upgrade to the latest version 2.0.54 ? 

Not sure if the builds are ok, but it does state that .NET Core 3 SDK is required, so we might need to upgrade the builds to install that if they are failing.

Reply:
we synced off line...updating caused other warnings..esp w/xamarin ios/android,